### PR TITLE
Add distributed

### PR DESCRIPTION
**Pull request by koverholt**

### CODE REVIEW

File: recipes/distributed/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    - distributed.http
+  commands:
+    - dcenter --help
+    - dscheduler --help
+    - dcluster --help
+    - dworker --help
+
+about:
+  home: http://distributed.readthedocs.io/en/latest/
+  license: BSD
```
Comment (jakirkham) [inline recipes/distributed/meta.yaml]: Please specify the type of BSD.

**Comment by jakirkham** [comment]
One comment. Once that is resolved this should be good to go.

File: recipes/distributed/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  md5: 98c6ec894386a93ffd16718d04b79671
+
+build:
+  number: 0
+  entry_points:
+    - dcenter=distributed.cli.dcenter:go
+    - dscheduler=distributed.cli.dscheduler:go
+    - dcluster=distributed.cli.dcluster:start
+    - dworker=distributed.cli.dworker:go
+  script: python setup.py install
```
Comment (jakirkham) [inline recipes/distributed/meta.yaml]: Please change to `python setup.py install --single-version-externally-managed --record=record.txt`.

**Comment by jakirkham** [comment]
Couple comments. Once they are resolved this should be good to go.

**Comment by koverholt** [comment]
I made the two changes you suggested. Thanks!

**Comment by 183amir** [comment]
LGTM and thanks @koverholt .

**Comment by jakirkham** [comment]
Thanks @koverholt.

File: recipes/distributed/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    - dworker --help
+
+about:
+  home: http://distributed.readthedocs.io/en/latest/
+  license: BSD 3-Clause
+  summary: 'Distributed computing'
+
+extra:
+  recipe-maintainers:
+    - mrocklin
```
Comment (jakirkham) [inline recipes/distributed/meta.yaml]: Sorry, should have done this before. Could you please confirm that you are happy to be a maintainer, @mrocklin?

Comment (mrocklin) [inline recipes/distributed/meta.yaml]: Confirmed that I am happy to be a maintainer.

Comment (jakirkham) [inline recipes/distributed/meta.yaml]: Awesome. Thanks @mrocklin. 😄

**Pull request merged**